internal/serving/grpcserver.go

```diff
@@ -40,13 +42,42 @@ func (gw *GrpcWrapper) AddService(handlerFunc func(*grpc.Server)) {
 	gw.serviceHandlerFuncs = append(gw.serviceHandlerFuncs, handlerFunc)
 }
 
+// AddProxy registers a reverse proxy from REST to gRPC when server is created.
 func (gw *GrpcWrapper) AddProxy(proxyHandlerFunc func(context.Context, *runtime.ServeMux, string, []grpc.DialOption) error) {
 	gw.proxyHandlerFunc = func(endpoint string) (*runtime.ServeMux, error) {
 		ctx := context.Background()
 		ctx, cancel := context.WithCancel(ctx)
 		defer cancel()
 
 		mux := runtime.NewServeMux()
+
+		type stubPattern struct {
```

Comment:
Why are we faking a real grpc server with a semi-mocked http server? You should be able to use the real server in the e2e tests and be good.